pangeo-notebook/packages.txt

```diff
@@ -113,80 +114,82 @@ colorspacious==1.1.2
 comm==0.2.2
 configobj==5.0.9
 contextily==1.6.2
-contourpy==1.3.0
-cryptography==40.0.1
+contourpy==1.3.1
+crc32c==2.7.1
+cryptography==43.0.1
 cuda-version==11.8
 cycler==0.12.1
-cytoolz==1.0.0
-dask==2024.11.0
-dask-core==2024.11.0
-dask-expr==1.1.17
+cytoolz==1.0.1
+dask==2024.11.2
+dask-core==2024.11.2
+dask-expr==1.1.19
 dask-gateway==2024.1.0
 dask-geopandas==0.4.2
 dask-glm==0.3.2
 dask-labextension==7.0.0
 dask-ml==2024.4.4
+datapi==0.1.2
 datashader==0.16.3
 dav1d==1.2.1
-debugpy==1.8.8
+debugpy==1.8.11
 decorator==5.1.1
 defusedxml==0.7.1
 descartes==1.1.0
-distributed==2024.11.0
+distributed==2024.11.2
 distro==1.8.0
-dnspython==2.6.1
+dnspython==2.7.0
 docopt-ng==0.9.0
 docrep==0.3.2
 docstring-to-markdown==0.15
-docutils==0.17.1
+docutils==0.18.1
 donfig==0.8.1.post1
 duckdb-cli==1.1.3
-earthaccess==0.11.0
-eccodes==2.38.3
+earthaccess==0.12.0
+eccodes==2.39.0
 email-validator==2.2.0
 email_validator==2.2.0
 entrypoints==0.4
-eofs==1.4.1
+eofs==2.0.0
 erddapy==2.2.3
-esmf==8.6.1
-esmpy==8.6.1
+esmf==8.7.0
+esmpy==8.7.0
 exceptiongroup==1.2.2
 executing==2.1.0
-fastapi==0.115.4
-fastapi-cli==0.0.5
-fasteners==0.17.3
+fastapi==0.115.6
+fastapi-cli==0.0.7
 fastjmd95==0.2.1
 fastprogress==1.0.3
 ffmpeg==7.1.0
 filelock==3.16.1
 findlibs==0.0.5
 flexcache==0.3
 flexparser==0.4
-flox==0.9.14
+flox==0.9.15
 fmt==11.0.2
-folium==0.18.0
+folium==0.19.4
 font-ttf-dejavu-sans-mono==2.37
 font-ttf-inconsolata==3.000
 font-ttf-source-code-pro==2.038
 font-ttf-ubuntu==0.83
 fontconfig==2.15.0
 fonts-conda-ecosystem==1
 fonts-conda-forge==1
-fonttools==4.54.1
+fonttools==4.55.3
 fqdn==1.5.1
 freeglut==3.2.2
 freetype==2.12.1
 freexl==2.0.0
 fribidi==1.0.10
 frozenlist==1.5.0
-fsspec==2024.10.0
+fsspec==2024.12.0
 future==1.0.0
+gast==0.4.0
 gcc==13.3.0
 gcc_impl_linux-64==13.3.0
 gcc_linux-64==13.3.0
-gcm_filters==0.5.0
-gcsfs==2024.10.0
-gdal==3.9.3
+gcm_filters==0.5.1
+gcsfs==2024.12.0
+gdal==3.10.0
```

Comment:
Bump from GDAL 3.9 to 3.10 too - https://github.com/OSGeo/gdal/blob/v3.10.0/NEWS.md#gdalogr-3100-release-notes